### PR TITLE
1916 remove select language from register form

### LINKED ISSUE #1916: 🔧 [Refactor] remove select language from register form
## 🔧 Refactor ticket
In the register Process tab, the language selection is there twice. The input field can be removed. 

FireShot Capture 1194 -  - localhost

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
remove select language from register form

### Issues
- fixes #1916 
 
 ### merge insert
- #1926 

https://user-images.githubusercontent.com/1324583/170006727-db75360c-bca5-4d7f-bd43-dd20f04d7d22.mp4

### CODE REVIEW

**Review by Hardy-Hart-am-Start — COMMENTED** [review]
nice

File: frontend/src/assets/scss/gradido-template.scss [hunk 1]
```diff
@@ -41,26 +41,49 @@ a,
 }
```
Comment (Tirokk) [inline frontend/src/assets/scss/gradido-template.scss:41]: Do this file changes belong to this PR?
I guess it belongs to #1927

**Review by Tirokk — APPROVED** [review]
This looks got to me! 🚀💫

Good catch !!! 👍🏼🙏🏼

File: frontend/src/pages/Register.vue [hunk 2]
```diff
@@ -84,7 +80,7 @@
               <b-button
                 type="submit"
                 :disabled="disabled"
-                :variant="disabled ? 'outline-gradido' : 'gradido'"
+                :variant="disabled ? 'gradido-disable' : 'gradido'"
```
Comment (Tirokk) [inline frontend/src/pages/Register.vue:83]: same same 😝

**Pull request merged**